src/store-tree/schema-placeholder.ls

```diff
@@ -48,10 +49,23 @@ is-map = ->
 
 get-type = ->
   | not is-placeholder it    => throw new Error "#{it} is not a placeholder"
-  | it is create-placeholder => SchemaType.ANY
   | otherwise                => it.type
 
 
+normalize = (input) ->
+  if input is create-placeholder
+    input!
+  else if input instanceof SchemaPlaceholder
+    input
+  else if input instanceof SchemaMap
+    input.child-schema = normalize input.child-schema
+    input
+  else if typeof! input is 'Object'
+    input |> Obj.map normalize
+  else
+    input
```

Comment:
Actually nope not possible as that error message has the path. This get called early on and we don't want to throw until we actually build the store